### ISSUE TITLE
"save all" does not appear in search results in some circumstances

### ISSUE THREAD

**Issue by putt1ck** [Zimbra-Community/owncloud-zimlet#220]
If:

conversation view is in use, and;
you search for X, and;
one of the emails in the results from the search has multiple attachments, and;
the thread contain that email has only one email, then...

Then "save all" does not appear as an option. Tested on multiple browsers and multiple installs of Zimbra with zimlet. Discovered by accident when testing by using search has:attachment.

**Comment by barrydegraaff** [comment]
Please add steps to reproduce, thanks.

**Comment by putt1ck** [comment]
Yeah I know I added the title before adding the description... Was all fixed 2 minutes later, but I guess Github had already emailed you by then.

**Comment by putt1ck** [comment]
To be clear, by "fixed" I mean I fixed my error and added details showing in what circumstances the bug arises. Or is this closed because you've already got a patch?

**Comment by barrydegraaff** [comment]
It is closed because there are no reproduction steps in this ticket, I thought you meant it was a cache issue.
